tests/ecosystem/deployment/test_acm.py

```diff
@@ -7,3 +9,6 @@
 
 def test_import_acm():
     import_clusters_with_acm()
+    driver = login_to_acm()
```

Comment:
this file is a placeholder, as it is not real tests but part of deployment, for this reason, i would suggest to create 1 function that summarize everything in acm.py file, and just call it from here

Reply:
I have created another test function for now which uses conftest, and this can be changed in the future if needed, to integrate it with other CLI functions.

Reply:
@AviadP Submariner creation is dependent on import_clusters_with_acm() function, so it can not be in a separate test function. I have reverted it back to test_import_acm.

Reply:
Any changes needed in the future should be part of another PR.